react/components/MultiSelect/README.md

```diff
@@ -0,0 +1,31 @@
+```js
+initialState = {
+  selected: ["Green", "Red"]
+};
+selectableList = [
+  "White",
+  "Black",
+  "Grey",
+  "Yellow",
+  "Red",
+  "Blue",
+  "Green",
+  "Brown",
+  "Pink",
+  "Orange",
+  "Purple",
+  "Dark-blue",
+  "Dark-red",
+  "Light-blue"
+];
+<MultiSelect
+  label="Colors"
+  onChange={selected => setState({ selected: [...selected] })}
+  onSearch={term =>
+    selectableList
+      .filter(tag => tag.toLowerCase().includes(term.toLowerCase()))
+      .filter(tag => !state.selected.includes(tag))
+  }
```

Comment:
só pra ver se eu entendi: esse `onSearch` é a funçao que vai fazer a busca dos itens? se for, pq isso nao é interno do componente?

Reply:
poderia imitar o dropdown, e passar um array de opções com `value` e `label`, e a busca ser feita no `label` e o valor retornado ser o `value`.

Reply:
Não deveria ser interno do componente porque o componente é "burro", ele não sabe buscar. Quem tem de explicar como buscar é quem está usando o componente.

Aliás, é uma questão que eu preciso levantar: sei que há necessidade de ter uma busca GraphQL, mas imagino que também existam casos simples onde temos uma quantidade pequena e limitada de opções. Sinto que seria bom ter ambos, mas tenho de ver a necessidade do pessoal quanto ao componente.

Gostei da ideia de value e label, estou pensando em como faria isso.